src/components/App/App.js

```diff
@@ -1,64 +1,59 @@
 import React from 'react'
 import Header from '../Header/Header'
 import MovieCard from '../MovieCard/MovieCard'
-import movieData from '../movieData/movieData'
 import InduvidualMovie from '../InduvidualMovie/InduvidualMovie'
 import './App.css';
-import { useState } from 'react';
-
-
+import { useState, useEffect } from 'react';
 
 function App() {
-const [movies, setMovies] = useState(movieData.movies);
-const [movieView, setMovieView] = useState(false);
-const [moviesView, setMoviesView] = useState(true);
-const [picked, setPicked] = useState();
 
-let filtered = movies.sort((a,b)=> a.title.localeCompare(b.title)).map(movie => <MovieCard poster={movie.poster_path} title={movie.title} id={movie.id} key={movie.id} findMovie={findMovie}/>)
+  async function fetchData() {
+    try {
+      let response = await fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+      setError(response.status)
+      let responseJson = await response.json();
+      return responseJson;
+    } catch (error) {
+      handleError()
+    }
+  }
+  useEffect(() => {
+    fetchData().then(
+      data => setMovies(data.movies)
+    )
+  })
+  function handleError() {
+    if (error !== '') {
+      throw new Error(`HTTP Error: ${error} -- Please try again`)
+    }
+  }
 
+  const [movies, setMovies] = useState([]);
+  const [error, setError] = useState('')
+  const [movieView, setMovieView] = useState(false);
+  const [moviesView, setMoviesView] = useState(true);
+  const [picked, setPicked] = useState();
 
+  let filtered = movies.map(movie => <MovieCard poster={movie.poster_path} title={movie.title} id={movie.id} key={movie.id} findMovie={findMovie}/>)
```

Comment:
Nice job keeping this logic pulled out of the return statement and storing in a variable instead.